Cangjie5.txt

```diff
@@ -5835,6 +5835,7 @@
 𧠘	buhlo
 𪾯	buhlo
 𥌝	buhmf
+𥍌	buhmf
```

Comment:
May be `buhxf`.

Reply:
`buhmf` is intentional: In the reference G-source glyph, the horizontal stroke is touching with 𦥑, though it should have been `buhxf` if the right component can be normalized to other 𤐫.

For the same reason we also have `𥌝	buhmf`.

Reply:
According to the Manual of the 5th generation of Cangjie , Both of these components are encoded as HX in Cangjie, regardless whether the horizontal stroke is touching with 𦥑 or not.

Reply:
According to the Manual of the 5th Generation of Cangjie, "𦥑" and "𦥑 touching with a horizontal stroke" are both encoded as "HX". So, "BUHMF" is unnecessary.